### PR TITLE
fix: improve error rendering 

### LINKED ISSUE #140: Improve Colour Rendering inside of error text
As a user, I'd like to be able to scan error messages quickly and take action on the error so that I can return to my original task.

I believe the WP standard is to have black text with red links for error messages:

Currently some error messages (consortial interface) are rendering in all red text:

Mixing red and black type to provide better overall hierarchy allows the user to focus on the specific errors and action items without being overwhelmed:

### PR DESCRIPTION
**Pull request by ho-man-chan**
Resolves: https://github.com/pressbooks/pressbooks-multi-institution/issues/140

This PR sets the html strong tag in error notices to be red (--pb-error-color).

Test:
1. Add invalid and/or already used emails in an institution's Email Domains field.

Expected Output:
image

### CODE REVIEW

**Comment by SteelWagstaff** [comment]
Rather than changing the display color of the `<strong>` tag itself, I think I'd prefer using a span with a class, and targeting that, but I'm ok with whatever approach the team feels is best and is willing to support.

File: resources/views/partials/errors/duplicate-domain.blade.php [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <li class="padding">
-	{!! sprintf(__('The email domain %s is already in use with %s. Please use a different address.', 'pressbooks-multi-institution'), $domain, $institution) !!}
+	{!! sprintf(__('The email domain %s is already in use with %s. <strong>Please use a different address.</strong>', 'pressbooks-multi-institution'), $domain, $institution) !!}
```
Comment (fdalcin) [inline resources/views/partials/errors/duplicate-domain.blade.php]: Is it possible to split the message so we don't have `<strong>` tags inside of the message?
```suggestion
	{!! sprintf(__('The email domain %s is already in use with %s.', 'pressbooks-multi-institution'), $domain, $institution) !!}
	<strong>{{ __('Please use a different address.', 'pressbooks-multi-institution') }}</strong>
```

**Review by fdalcin — CHANGES_REQUESTED** [review]
LGTM and works as expected. I suggested a small change to avoid having `strong` tags inside the translation string.

In addition, the duplicate institution name error message does not have a `strong` tag for the fix instruction.

**Comment by fdalcin** [comment]
> Rather than changing the display color of the `\<strong\>` tag itself, I think I'd prefer using a span with a class, and targeting that, but I'm ok with whatever approach the team feels is best and is willing to support.

I like your suggestion better. In that case the only `strong` parts would be the invalid domains and duplicate domains.

**Comment by fdalcin** [comment]
I've pushed a friendly commit with updates to the error messages based on the discussion above, see below:

image

**Review by fdalcin — APPROVED** [review]
LGTM

**Pull request merged**